### DISCUSSION TITLE
Nx needs to run with bun.lock file

### DISCUSSION THREAD

**Discussion by ludicast** [nrwl/nx#29422] [Feature Requests]
The `bun.lock` format (text lockfile) is [a new style of bun lockfile](https://bun.sh/blog/bun-lock-text-lockfile). It enables dependabot to work on bun repos and is to be the default on 1.2.

**Comment by ludicast** [comment]
trying to address in https://github.com/nrwl/nx/pull/29423

**Comment by JamesHenry** [comment]
@ludicast For future reference this one should really be an issue I think, because it makes our `bun` support incomplete in the latest version of `bun`.

It will be resolved by https://github.com/nrwl/nx/pull/30064
